[Suggestion] Updating READ-ME

Mxrcon opened the pull request:
# [Suggestion] Updating READ-ME

Hey @abhi18av, I took some time to update the read-me using a template that i used to other projects, I would like to also implement a dag.png workflow image as this could be a directive for new users to the pipeline,

# PR content
* Read-me 

# Ending

Feel free to contact me if you have any doubt, critic or suggestion. I'm at your disposal to make any changes that you request.

Hope we have a mege!
Regards, Davi

abhi18av commented:
@Mxrcon , thanks for taking the time to make this effort. 

I have updated the code quite a bit and once that is all set, would it be possible for you to update this PR please?

Mxrcon commented:
@abhi18av Yes! Sure, as soon as i update this PR I'll mention you for a new review.

Mxrcon commented:
@abhi18av  Hey, I've just updated the commits, i took some time to update the readme and generated a new dag.png. I have some troubles when running stubs for this pipeline, I'm at your disposal to review the stubs and find the causes of this errors, would you like it? I've also updated the sutb_params.yml
